docs/tutorials/lut.rst

```diff
@@ -85,6 +89,10 @@ Fast-marching method
 ####################
```

Comment:
I can’t find a way to make this comment in a sensible place - sorry. I think the read_vmodel section should be moved up here before either of the 1d methods.